cli/groups.go

```diff
@@ -0,0 +1,166 @@
+	"github.com/spf13/cobra"
+)
+
+var cmdGroups = []cobra.Command{
+	cobra.Command{
+		Use:   "create",
+		Short: "create <JSON_group> <user_auth_token>",
+		Long: `Creates new group
+		JSON_group: 
+		{
+			"Name":<group_name>,
+			"Description":<description>,
+			"ParentID":<parent_id>,
+			"Metadata":<metadata>,
+		}
+		Name - is unique group name
+		ParentID - ID of a group that is a parent
+		to the creating group
+		Metadata - JSON structured string`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 2 {
+				logUsage(cmd.Short)
+				return
+			}
+
+			var group mfxsdk.Group
+			if err := json.Unmarshal([]byte(args[0]), &group); err != nil {
+				logError(err)
+				return
+			}
+
+			id, err := sdk.CreateUsersGroup(group, args[1])
+			if err != nil {
+				logError(err)
+				return
+			}
+			logCreated(id)
+		},
+	},
+	cobra.Command{
+		Use:   "get",
+		Short: "get [all | children <group_id> | group_id] <user_auth_token>",
+		Long: `Get all users groups or group by id.
+		all - lists all groups
+		all <group_id> - lists all children groups of <group_id>
+		<group_id> - shows group with provided group ID`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) < 2 {
+				logUsage(cmd.Short)
+				return
+			}
+
+			if args[0] == "all" {
+				l, err := sdk.UsersGroups(args[1], uint64(Offset), uint64(Limit), "")
+				if err != nil {
+					logError(err)
+					return
+				}
+				logJSON(l)
+				return
+			}
+
+			if args[0] == "children" {
+				l, err := sdk.UsersGroups(args[2], uint64(Offset), uint64(Limit), args[1])
+				if err != nil {
+					logError(err)
+					return
+				}
+				logJSON(l)
+				return
+			}
+			t, err := sdk.UsersGroup(args[0], args[1])
+			if err != nil {
+				logError(err)
+				return
+			}
+
+			logJSON(t)
+		},
+	},
+	cobra.Command{
+		Use:   "add",
+		Short: "add <user_id> <group_id> <user_auth_token>",
+		Long:  `add user to a group.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 3 {
+				logUsage(cmd.Short)
+				return
+			}
+			if err := sdk.AddGroupUser(args[0], args[1], args[2]); err != nil {
+				logError(err)
+			}
+		},
+	},
+	cobra.Command{
+		Use:   "remove",
+		Short: "remove <user_id> <group_id> <user_auth_token>",
+		Long:  `remove user from a group.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 3 {
+				logUsage(cmd.Short)
+				return
+			}
+			if err := sdk.RemoveGroupUser(args[0], args[1], args[2]); err != nil {
+				logError(err)
+			}
+		},
+	},
+	cobra.Command{
+		Use:   "delete",
+		Short: "unassign <group_id> <user_auth_token>",
+		Long:  `Delete users group.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 2 {
+				logUsage(cmd.Short)
+				return
+			}
+			if err := sdk.DeleteUsersGroup(args[0], args[1]); err != nil {
+				logError(err)
+			}
+		},
+	},
+	cobra.Command{
+		Use:   "users",
+		Short: "users <group_id> <user_auth_token>",
+		Long:  `Lists users in users group.`,
```

Comment:
Be consistent in the use of capital letters in descriptions

Reply:
`users in users group.` could be `group users`